pkg/ccl/importccl/testdata/mysqldump/everything-schema.sql

```diff
@@ -0,0 +1,74 @@
+-- MySQL dump 10.13  Distrib 5.7.22, for osx10.13 (x86_64)
+--
+-- Host: localhost    Database: cockroachtestdata
+-- ------------------------------------------------------
+-- Server version	5.7.22
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `everything`
+--
+
+DROP TABLE IF EXISTS `everything`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `everything` (
+  `i` int(11) NOT NULL,
+  `c` char(10) DEFAULT NULL,
+  `s` varchar(100) DEFAULT NULL,
+  `tx` text,
+  `bin` binary(100) DEFAULT NULL,
+  `vbin` varbinary(100) DEFAULT NULL,
+  `bl` blob,
+  `dt` datetime DEFAULT NULL,
+  `d` date DEFAULT NULL,
+  `ts` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
```

Comment:
do we need to error or something because of the `on update`?

Reply:
I think we need to doc it as one of the migration concerns, but if we're strict here and error on it (or require some option or other user intervention), I think we'll reject most schemas or make the process harder.

I'd be more inclined to do that if I thought it would be preventing accidental mistakes / bugs, but  given just how different the rest of the SQL semantics are (for example, case sensitivity), I think there's already going to be a migration process required well beyond just the data import, so I think we can lean towards being easier / a bit more lenient in the import step.

Reply:
Agreed.